### PR TITLE
Added the Count method

### PR DESCRIPTION
**Pull request by rizkybiz**
Added the Count method to Histogram in order to have continuity between Histogram support for different languages.

### CODE REVIEW

**Comment by maier** [comment]
lgtm

File: circonusllhist.go [hunk 1]
```diff
@@ -456,6 +456,19 @@ func (h *Histogram) Mean() float64 {
...
 
+//Count returns the number of recorded values.
+func (h *Histogram) Count() uint64 {
+	if h.useLocks {
+		h.mutex.RLock()
+		defer h.mutex.RUnlock()
+	}
+	var count uint64
+	for _, bin := range h.bvs[0:h.used] {
+		count += bin.count
```
Comment (esproul) [inline circonusllhist.go:467]: In other languages we've done an early-out for NaN bins; not sure if that's needed with this approach.

**Review by esproul — APPROVED** [review]
One question, but LGTM.

Might also add `BinCount` to match up with the C lib's `hist_bucket_count()`. We have that right in the Histogram struct as `used`.

File: circonusllhist.go [hunk 2]
```diff
@@ -456,6 +456,32 @@ func (h *Histogram) Mean() float64 {
...
+	for _, bin := range h.bvs[0:h.used] {
+		if bin.isNaN() {
+			continue
+		}
+		count += bin.count
+	}
+	return count
+}
+
+//BinCount returns the number of bins.
```
Comment (esproul) [inline circonusllhist.go:475]:
```suggestion
//BinCount returns the number of used bins.
```

**Review by esproul — APPROVED** [review]
One nit for a comment but LGTM

File: circonusllhist.go (hunk 2 above)
Comment (rizkybiz) [reply circonusllhist.go]: Latest commit resolves this

**Review by maier — APPROVED** [review]
lgtm

**Pull request merged**